config.json

```diff
@@ -70,6 +70,8 @@
     "product_list_display_mode": "grid",
     "logo-position": "center",
     "logo_size": "250x100",
+    "logo_fontSize": 12,
```

Comment:
We'll need to add some code to the template and/or the css to make use of this new value too.